resource/ui/ts/models/proto.ts

```diff
@@ -113,6 +155,32 @@ module Models {
             available_range_count: number;
         }
 
+        /**
+         * Status is the common interface shared by NodeStatus and StoreStatus.
+         */
+        export interface Status{
```

Comment:
A little bit. I prefer explicit interfaces.  But at least with TS and Go, we get static analysis to tell us when we're wrong.